apps/easy-messe/components/layout/Header/SideBar/SideBar.tsx

```diff
@@ -3,23 +3,23 @@ import { INavItem } from "../Header";
 import { NavItem } from "../navitem";
 import LanguageSwapper from "../../LanguageSwapper";
 import Image from "next/image";
-import LogoEasy from '../../../../assets/LogoEasyMass.png'
 import { useRouter } from "next/router";
 import { useIntl } from "react-intl";
+import { Dispatch, SetStateAction } from "react";
 
 
 interface sideBarProps {
     open: boolean,
-    toggleDrawer: (newOpen: boolean) => void,
+    closeSideBar: Dispatch<SetStateAction<boolean>>,
```

Comment:
It's just a function.

```suggestion
    closeSideBar: ()=>void,
```

Reply:
When I type this function as you review, i encountered an error telling me that this function receive 0 params but got one. It's true because when calling it, I set a boolean variable on it. So i thing that have a parameter on it may resoltve this issue.